### PR TITLE
Hide cursor after 2s in fullscreen

### PR DESCRIPTION
**Pull request by ngoldman**

### CODE REVIEW

**Comment by ngoldman** [comment]
Missed commit from `dc/state` branch. Rebasing.

File: renderer/index.js [hunk 1]
```diff
@@ -360,7 +364,10 @@ function setDimensions (dimensions) {
   var width = Math.floor(dimensions.width * scaleFactor)
   var height = Math.floor(dimensions.height * scaleFactor)
 
-  height += HEADER_HEIGHT
+  // Video player header only shows in OSX where it's part of the title bar. See app.js
+  if (process.platform === 'darwin') {
+    height += HEADER_HEIGHT
+  }
```
Comment (feross) [inline renderer/index.js]: good catch

File: renderer/views/player.js [hunk 2]
```diff
@@ -23,10 +23,16 @@ function Player (state, dispatch) {
     state.video.duration = videoElement.duration
   }
 
+  // When in fullscreen, hide player controls if the mouse stays still for a while
+  //console.log('STATIONARY FOR ' + new Date().getTime() - state.video.mouseStationarySince)
```
Comment (feross) [inline renderer/views/player.js]: We should hide the player controls if the mouse is still, even when not in fullscreen.

YouTube, Quicktime do the same.

**Comment by ngoldman** [comment]
Looks like after rebase it was already in master :stuck_out_tongue_closed_eyes:

**Comment by feross** [comment]
Heh, I'm confused. So @dcposch made these changes? I left some inline comments, in any case.

**Comment by ngoldman** [comment]
I saw a branch that looked like it had a commit that didn't get merged and it looked like good stuff so I thought I'd try to get it in, not realizing it had already been pushed to master.

¯\_(ツ)_/¯

**Comment by feross** [comment]
Okay, got it. Thanks for explaining :+1:

**Comment by dcposch** [comment]
Yeah I did, late last night. I'll make PRs from now on so you can see what I'm up to.

**Pull request closed without merging**